codalab/lib/bundle_cli.py

```diff
@@ -1277,6 +1277,8 @@ def do_download_command(self, args):
 
         # Do the download.
         target_info = client.fetch_contents_info(bundle_uuid, subpath, 0)
+        bundle_uuid = target_info.get('resolved_uuid', bundle_uuid)
+        resolved_subpath = target_info.get('resolved_path', subpath)
```

Comment:
just use `subpath` to make the naming a bit more symmetric (since `bundle_uuid` gets overridden)

Reply:
I didn't override the variable `subpath` here because it is used to print the download source to the console in the CLI. If we overwrote it, `cl down <bundle-1>/<dep-key>/<path>` would print `<dep-bundle>/<path>` in the download progress. But on second thought, that seems reasonable too (I just didn't want to change what download printed at first)